src/Simplex/Chat.hs

```diff
@@ -7006,59 +7165,33 @@ sendGroupMessage user gInfo members chatMsgEvent = do
       currentTs <- liftIO getCurrentTime
       withStore' $ \db -> updateUserMemberProfileSentAt db user gInfo currentTs
 
-type GroupSndResultData = (([Either ChatError ([Int64], PQEncryption)], [(GroupMember, Connection)]), ([Either ChatError ()], [GroupMember]), [GroupMember])
-
 data GroupSndResult = GroupSndResult
-  { sentTo :: [GroupMember],
-    pending :: [GroupMember],
+  { sentTo :: [(GroupMemberId, Either ChatError [MessageId], Either ChatError ([Int64], PQEncryption))],
```

Comment:
It's a different GroupSndResult from previous. This one is closer to GroupSndResultData in purpose. GroupSndResultData is a worse name, so new code uses name from one and purpose from other. Diff is bad regardless of its position.